gridstatus/pjm.py

```diff
@@ -183,7 +185,10 @@ def get_load_forecast(self, date, verbose=False):
 
     def get_pnode_ids(self):
         data = {
-            "fields": "effective_date,pnode_id,pnode_name,pnode_subtype,pnode_type,termination_date,voltage_level,zone",
+            "fields": (
+                "effective_date,pnode_id,pnode_name,pnode_subtype,               "
```

Comment:
did this string get converted properly?

Reply:
Fixed:
https://github.com/kmax12/gridstatus/pull/114/commits/740be4bc98a811bd22bf9730fbf77caa700fff6c